src/core/src/op/abs.cpp

```diff
@@ -69,6 +91,27 @@ bool Abs::has_evaluate() const {
         return false;
     }
 }
+
+bool Abs::evaluate_lower(TensorVector& output_values) const {
+    return ov::op::abs::evaluate_bound(this, get_input_tensor(0), output_values, true);
+}
+
+bool Abs::evaluate_upper(TensorVector& output_values) const {
+    return ov::op::abs::evaluate_bound(this, get_input_tensor(0), output_values, false);
+}
+
+bool Abs::evaluate_symbol(TensorSymbolVector& output_symbols) const {
+    const auto& non_negative = ov::util::to_vector<bool>(ov::util::ge(get_input_tensor(0).get_lower_value(), 0));
+    const auto& input_symbols = get_input_tensor(0).get_value_symbol();
+    if (non_negative.empty() || input_symbols.size() != non_negative.size())
+        return false;
```

Comment:
```suggestion
    const auto& non_negative = ov::util::to_vector<bool>(ov::util::ge(get_input_tensor(0).get_lower_value(), 0));
    const auto& input_symbols = get_input_tensor(0).get_value_symbol();
    if (!non_negative || input_symbols.size() != non_negative.size())
        return false;
```
tensor can be check without cast to vector